CSharpAdvanced/CSharpAdvanced/Rose.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CSharpAdvanced
+{
+    public class Rose : Flower
+    {
+        private string _name;
+        private string _color;
+        private double _price;
+
+        public override string Name { get => _name; set => _name = value; }
```

Comment:
hmm, you overrided values, but new implementation didn't give anything new to existing implementation. You can add new constructor here of add custom property :)